org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationMatching.cs

```diff
@@ -0,0 +1,718 @@
+            }
+
+            //Add LastNames
+            worksheet.Cells[row, 3].Value = contact.properties.lastname;
+            worksheet.Cells[row, 4].Value = person.LastName;
+            //Color cells if they match 
+            if ( CustomEquals(contact.properties.lastname, person.LastName) )
+            {
+                worksheet = ColorCell(worksheet, row, 3);
+                worksheet = ColorCell(worksheet, row, 4);
+            }
+
+            //Add Emails
+            worksheet.Cells[row, 5].Value = contact.properties.email;
+            worksheet.Cells[row, 6].Value = person.Email;
+            //Color cells if they match
+            if ( CustomEquals(contact.properties.email, person.Email) )
+            {
+                worksheet = ColorCell(worksheet, row, 5);
+                worksheet = ColorCell(worksheet, row, 6);
+            }
+
+            //Add Phone Numbers
+            var num = person.PhoneNumbers.FirstOrDefault(pn => pn.Number == contact.properties.phone);
+            worksheet.Cells[row, 7].Value = contact.properties.phone;
+            worksheet.Cells[row, 8].Value = num != null ? num.Number : "No Matching Number";
+            //Color cells if they match
+            if ( num != null && CustomEquals(contact.properties.phone, num.Number) )
+            {
+                worksheet = ColorCell(worksheet, row, 7);
+                worksheet = ColorCell(worksheet, row, 8);
+            }
+
+            //Add Connection Statuses
+            worksheet.Cells[row, 10].Value = person.ConnectionStatusValue;
+
+            //Add links
+            worksheet.Cells[row, 11].Value = "https://app.hubspot.com/contacts/6480645/contact/" + contact.id;
+            worksheet.Cells[row, 12].Value = "https://rock.lakepointe.church/Person/" + person.Id;
+
+            //Add Created Dates
+            if ( !String.IsNullOrEmpty(contact.properties.createdate) )
+            {
+                DateTime hubspotVal;
+                if ( DateTime.TryParse(contact.properties.createdate, out hubspotVal) )
+                {
+                    worksheet.Cells[row, 13].Value = hubspotVal.ToString("MM/dd/yyyy");
+                }
+            }
+            worksheet.Cells[row, 14].Value = person.CreatedDateTime.Value.ToString("MM/dd/yyyy");
+
+            //Add Modified Dates
+            if ( !String.IsNullOrEmpty(contact.properties.lastmodifieddate) )
+            {
+                DateTime hubspotVal;
+                if ( DateTime.TryParse(contact.properties.lastmodifieddate, out hubspotVal) )
+                {
+                    worksheet.Cells[row, 15].Value = hubspotVal.ToString("MM/dd/yyyy");
+                }
+            }
+            worksheet.Cells[row, 16].Value = person.ModifiedDateTime.Value.ToString("MM/dd/yyyy");
+
+            //Add Rock Id
+            worksheet.Cells[row, 17].Value = person.Id;
+
+
+            return worksheet;
+        }
+
+        private bool CustomEquals(string a, string b)
```

Comment:
Could we rename `CustomEquals` to something like `HasValueAndEquals` or something else that describes what it does. Methods should have names that describe what they do in order to make code more readable.